ignite3/src/main/java/site/ycsb/db/ignite3/IgniteAbstractClient.java

```diff
@@ -115,9 +115,17 @@ public abstract class IgniteAbstractClient extends DB {
 
   /**
    * Used to choose storage engine (e.g., 'aipersist' or 'rocksdb').
+   * @deprecated Removed in <a href="https://ggsystems.atlassian.net/browse/IGN-23905">IGN-23905</a>
    */
+  @Deprecated
   protected static String dbEngine;
 
+  /**
+   * Used to choose storage profiles separated by comma
+   * (e.g., 'default', 'default_aimem', 'default_aipersist,default_rocksdb', etc.).
```

Comment:
```suggestion
   * (e.g., 'default', 'default_aimem', 'default_aipersist', 'default_rocksdb', etc.).
```

Reply:
Actually, I wanted to give an example of several storage profiles sent in one line.
As per [documentation of Zone declaration](https://docs.google.com/document/d/1VHNybikLk4I-ULJjkupuGIN2D9kxF7obRTQ7cNwBeLo/edit#bookmark=kix.h0t8uyplp4zr):
```
create zone z1 with storage_profiles='lru_rocks,clock_aipersist'; 
```